bench/README.md

```diff
@@ -1,60 +1,175 @@
+
+6. View the benchmarking reports at your specified **benchmark\_report_path**.
+
+## Interpreting the reports
+
+Metrics from your benchmarking run will be located at 
+**benchmark\_report_path**.
+
+The file starts with documentation of the configuration details you supplied:
+
+    ---Configuration---
+     "cifar_data_path":"<cifar_data_path>",
+     "num_threads":"<num_threads>",
+     "num_batches":"<num_batches>",
+     "request_batch_size":"<request_batch_size>",
+     "request_batch_delay_micros":"<request_batch_delay_micros>",
+     "poisson_delay":"<true/false>",
+     "prevent_cache_hits":"<true/false>",
+     "latency_objective":"<latency_objective>",
+     "benchmark_report_path":"<benchmark_report_path>",
+     "report_delay_seconds":"<report_delay_seconds>",
+     "model_name":"<model_name>",
+     "model_version":"<model_version>"
+    ------------------- 
+
+After, metrics are shared by time window. Recall that each window is of length **report\_delay_seconds** (unless you set it to `-1`, in which case there will only be one window that captures the length of the whole run):
+
+
+  <Window 1 start time> - <Window 1 end time>: {
+    <Metrics captured in Window 1>
+  }
+  <Window 2 start time> - <Window 2 end time>: {
+    <Metrics captured in Window 1>
+  }
+  ...
   
-  Otherwise, execute
-  ```sh
-  ./src/benchmarks/end_to_end_bench
-  ```
-  and specify the values of the attributes enumerated in the **Optional Configuration Files** section above. 
+There are several metrics captured in each window. The notable ones are listed below:
+
+Under `ratio_counters`:
+
+- `bench:cifar_bench:default_prediction_ratio`: The fraction of requests to which Clipper responded with a default prediction (bypassing querying the model). This ratio should only be greater than 0 when requests are sent at a rate faster than the Clipper system and your model can serve them.
+- `model:<model_name>:<model_version>:cache_hit_ratio`: The fraction of requests that were served by cache lookups instead of model-generated predictions.
+
+Under `meters`:
+
+- `model:<model_name>:<model_version>:prediction_throughput`: The rate at which your model-container is serving requests
+- `bench:cifar_bench:request_throughput`: The rate at which the benchmarking tool is sending requests.
+- `bench:cifar_bench:prediction_throughput`: The rate at which the whole Clipper system is serving requests.
+
+Under `histograms` (each of these entries lists the unit of measurement, number of datapoints, min, max, mean, standard deviation, and p50, p95, and p99 values):
+
+- `bench:cifar_bench:prediction_latency`: Statistics on latency measured from when the benchmarking script sends a request to when it receives a response.
+- `model:<model_name>:<model_version>:prediction_latency`: Statistics on latency measured from when Clipper sends datapoints to your model-container to when it receives a response. The value will always be lower than that of the former metric.

```

Comment:
Remove the last sentence under this bullet. The model prediction latency can definitely be higher than prediction latency, because predictions will return early if the SLO is set to be lower than the model latency.